src/plugins/intel_gpu/src/graph/impls/ocl_v2/col2im.cpp

```diff
@@ -0,0 +1,153 @@
+// Copyright (C) 2025 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "col2im.hpp"
+
+#include "col2im_inst.h"
+#include "common_utils/dispatch_utils.hpp"
+#include "intel_gpu/graph/kernel_impl_params.hpp"
+#include "primitive_ocl_base.hpp"
+#include "utils/kernel_generator.hpp"
+
+namespace ov::intel_gpu::ocl {
+namespace {
+
+static std::vector<size_t> get_origin_size(const kernel_impl_params& params) {
```

Comment:
nit: `original_size` or `orig_size`. `origin_size` sounds weird.

Reply:
Fixed to orig_size